config/initializers/jasmine_rails.rb

```diff
@@ -0,0 +1,11 @@
+if defined?(JasmineRails)
```

Comment:
Do we need this file / condition?

It seems a bit strange that we apply this only when we have JasmineRails defined when it's available in both dev and test meaning this would, if I'm understanding correct, apply to every single request in dev and test?